tap_zoom/client.py

```diff
@@ -57,16 +74,16 @@ def refresh_access_token(self):
 
         self.__access_token = data['access_token']
         self.__refresh_token = data['refresh_token']
-
-        self.__expires_at = datetime.utcnow() + \
+        self.__expires_at = now() + \
             timedelta(seconds=data['expires_in'] - 10) # pad by 10 seconds for clock drift
 
-        ## refresh_token changes every call to refresh
-        with open(self.__config_path) as file:
-            config = json.load(file)
-        config['refresh_token'] = data['refresh_token']
-        with open(self.__config_path, 'w') as file:
-            json.dump(config, file, indent=2)
+        if not self.dev_mode:
+            update_config_keys = {
+                                    "refresh_token":self.__refresh_token,
+                                    "access_token":self.__access_token,
+                                    "expires_in": strftime(self.__expires_at)
```

Comment:
Storing `expires_in` might not be needed because if the user re-auths the connection after u cloned the connection then u have both `access_token` and `expires_in` in config which would become invalid as the user re-auth. So, just validating the `access_token` should be fine.checkout this [pr_comment](https://github.com/singer-io/tap-deputy/pull/3#discussion_r991573049)